internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -246,6 +246,39 @@ func (a ApplicationSnapshotImage) ValidateAttestationSyntax(ctx context.Context)
 	return EV003.CausedBy(errors.New(msg))
 }
 
+// FilterMatchingAttestations ignores attestations that do not have a matching subject.
+func (a *ApplicationSnapshotImage) FilterMatchingAttestations(ctx context.Context) {
+	filteredAttestations := make([]oci.Signature, 0, len(a.attestations))
+	for _, attestation := range a.attestations {
+		_, statement, err := statementFrom(ctx, attestation)
+		if err != nil {
+			log.Debugf("Unable to extract statement from attestation: %v", err)
```

Comment:
Oh shoot. I just realized there should be a `continue` here. I'll see if I can add a unit test to capture this case.

Reply:
Done.